src/ClientBuilder.php

```diff
@@ -125,7 +117,16 @@
      */
     public function __construct(array $options = [])
     {
-        $this->configuration = new Configuration($options);
+        $this->options = new Options($options);
+
+        if (null === $this->options->getIntegrations()) {
+            $this->integrations = [];
```

Comment:
What about initializing the variable where it's declared with an empty `array` and invert this `if` statement so that it gets overwritten if needed?

Reply:
We do not gain anything from this, the if does not really change.